Tasks/HelmDeployV0/src/helm.ts

```diff
@@ -73,6 +73,7 @@ async function run() {
         command: command,
         jobId: tl.getVariable('SYSTEM_JOBID')
     };
+    var failOnStderr = tl.getBoolInput("failOnStderr", true);
```

Comment:
it is not mandatory input so keep false.

Increase task.json minor version to 165 and pathversion to 0.

Reply:
fixed both